packages/driver/src/dom/elements/complexElements.ts

```diff
@@ -290,12 +290,6 @@ export const isScrollable = ($el) => {
     return false
```

Comment:
This file's changes involve undoing https://github.com/cypress-io/cypress/pull/27860